shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -153,11 +153,13 @@ def compress_with_zip(file_paths: list, compressed_file_name: str):
     Returns:
     None
     """
-    with ZipFile(compressed_file_name, "w") as zip_file:
+
+    with ZipFile(compressed_file_name, "w", ZIP_DEFLATED) as zip_file:
         for file in file_paths:
-            file = clean_folder_name(file.replace(os.getcwd(), ""))
-            zip_file.write(file)
+            print(file)
```

Comment:
Not sure if that was left here accidentally, but this should either be removed, or logged with more relevant information.

Reply:
forgot to remove it - good catch